Index: php.ini-production
===================================================================
--- php.ini-production.orig	2010-06-24 02:15:12.000000000 +0200
+++ php.ini-production	2010-08-03 06:31:20.319461000 +0200
@@ -781,7 +781,7 @@ default_mimetype = "text/html"
 ;;;;;;;;;;;;;;;;;;;;;;;;;
 
 ; UNIX: "/path1:/path2"
-;include_path = ".:/php/includes"
+include_path = ".:/usr/share/php5:/usr/share/php5/PEAR"
 ;
 ; Windows: "\path1;\path2"
 ;include_path = ".;c:\php\includes"
@@ -1189,7 +1189,7 @@ mysql.allow_local_infile = On
 
 ; Allow or prevent persistent links.
 ; http://php.net/mysql.allow-persistent
-mysql.allow_persistent = On
+mysql.allow_persistent = Off
 
 ; If mysqlnd is used: Number of cache slots for the internal result set cache
 ; http://php.net/mysql.cache_size
@@ -1252,7 +1252,7 @@ mysqli.max_persistent = -1
 
 ; Allow or prevent persistent links.
 ; http://php.net/mysqli.allow-persistent
-mysqli.allow_persistent = On
+mysqli.allow_persistent = Off
 
 ; Maximum number of links.  -1 means no limit.
 ; http://php.net/mysqli.max-links
@@ -1474,7 +1474,7 @@ session.save_handler = files
 ; where MODE is the octal representation of the mode. Note that this
 ; does not overwrite the process's umask.
 ; http://php.net/session.save-path
-;session.save_path = "/tmp"
+session.save_path = "/var/lib/php5"
 
 ; Whether to use cookies.
 ; http://php.net/session.use-cookies
@@ -1590,14 +1590,14 @@ session.referer_check =
 
 ; How many bytes to read from the file.
 ; http://php.net/session.entropy-length
-session.entropy_length = 0
+session.entropy_length = 32
 
 ; Specified here to create the session id.
 ; http://php.net/session.entropy-file
 ; On systems that don't have /dev/urandom /dev/arandom can be used
 ; On windows, setting the entropy_length setting will activate the 
 ; Windows random source (using the CryptoAPI)
-;session.entropy_file = /dev/urandom
+session.entropy_file = /dev/urandom
 
 ; Set to {nocache,private,public,} to determine HTTP caching aspects
 ; or leave this empty to avoid sending anti-caching headers.
@@ -1628,7 +1628,7 @@ session.use_trans_sid = 0
 ; the hash extension. A list of available hashes is returned by the hash_algos()
 ; function.
 ; http://php.net/session.hash-function
-session.hash_function = 0
+session.hash_function = sha256
 
 ; Define how many bits are stored in each character when converting
 ; the binary hash data to something readable.
